### PR TITLE
Scroll images in project issues separately from the remaining issue

### PR DESCRIPTION
**Pull request by Simandara**
As discussed in #31667 & #26561, when a card on a Project contains images, they can overflow the card on its containing column. This aims to fix this issue via snapping scrollbars.

---
Issue #31667 is open to discussion as there should be room for improvement.

### CODE REVIEW

**Comment by silverwind** [comment]
Hmm that seems like a lot of CSS properties and I'm not sure all are neccessary.

What is needed to reproduce it? Just issue with many images in it? Maybe you could create a reproduction on https://demo.gitea.com/ so I can take a closer look at that CSS.

**Comment by SimonPistache** [comment]
@silverwind sure have a look [here](https://demo.gitea.com/SimonPistache/test-project-card-with-images/projects/54)

**Comment by SimonPistache** [comment]
@silverwind It should be easier to read now.

I did the following:
 - `align-items: stretch;` and ` overflow: scroll;` manage the position of images in a card and avoid overflowing out of it
 -  `cursor: default;   scroll-snap-type: x mandatory; scroll-snap-align: center;` helps scrolls in images "snaps" so an image is centred (like a carrousel), this is optional
 - `aspect-ratio: 1;` come from [this thread](https://github.com/go-gitea/gitea/pull/26561#discussion_r1298819277), but I have not investigate further if it require a *max-width* with the *max-height* discussed in this topic.

File: web_src/css/features/projects.css [hunk 1]
```diff
@@ -70,15 +70,20 @@
 .card-attachment-images {
   display: inline-block;
   white-space: nowrap;
-  overflow: hidden;
+  overflow: scroll;
+  cursor: default;
```
Comment (delvh) [inline web_src/css/features/projects.css:74]: Isn't the default cursor used already?

**Review by delvh — APPROVED** [review]

File: web_src/css/features/projects.css (hunk 1 above)
Comment (Simandara) [reply web_src/css/features/projects.css:74]: On cards the cursor is a grabbing hand to clue about dragging them around. This reset the cursor hover the image area to help with the scrolling part, though it is still a draggable area. I could have used `cursor: unset`. We can remove it, it's not necessary.

**Review by lunny — APPROVED** [review]

**Comment by yp05327** [comment]
Please add `fix #31667` in the description. PR is not linked to the issue.

**Pull request merged**